source/retrieve-data.txt

```diff
@@ -0,0 +1,196 @@
+.. _java-rs-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, search, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to retrieve
+data from a MongoDB collection by using read operations. You can call the
+``find()`` method to retrieve documents that match a set of criteria.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
+
+.. _java-rs-retrieve-find:
+
+Find Documents
+--------------
+
+The {+driver-short+} includes one method for retrieving documents from a collection:
+``find()``.
+
+This method takes a **query filter** and returns one or more matching documents.
+A query filter is an object that specifies the documents you want to retrieve in
+your query. 
+
+.. TODO: uncomment when specify query is merged in
+.. To learn more about query filters, see :ref:`java-rs-specify-query`.
+
+.. _java-rs-retrieve-find-one:
+
+Find One Document
+~~~~~~~~~~~~~~~~~
+
+To find a single document in a collection, call the ``find()`` method and pass a query
+filter that specifies the criteria of the document you want to find. Then, chain
+the ``first()`` method to find a single document. If more than one document matches the query
+filter, this construct returns the *first* matching document from the retrieved
+results. If no documents match the query filter, the method returns
+``None``.
+
+.. tip::
+   
+   The ``find().first()`` construct is useful when you know there's only one matching document,
+   or you're only interested in the first match.  
```

Comment:
I: [placement of "only"](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#o:~:text=Place%20only%20preceding)

```suggestion
   or you're interested only in the first match.  
```